src/__tests__/CodePen.js

```diff
@@ -7,28 +8,45 @@ cases(
     expect(shouldTransform(url)).toBe(valid);
   },
   {
-    'direct url': {
-      url: 'https://codepen.io/team/codepen/pen/PNaGbb',
-      valid: true,
+    'non-CodePen url': {
+      url: 'https://not-a-codepen-url.com',
+      valid: false,
+    },
+    'CodePen team page': {
+      url: 'https://codepen.io/team/codepen',
+      valid: false,
+    },
+    'CodePen user page': {
+      url: 'https://codepen.io/MichaelDeBoey',
+      valid: false,
+    },
+    'random CodePen page': {
+      url: 'https://codepen.io/random-page',
+      valid: false,
     },
-    'embed url': {
+    'CodePen blog': {
+      url: 'https://blog.codepen.io',
+      valid: false,
+    },
+    'Pen embed url': {
       url: 'https://codepen.io/team/codepen/embed/PNaGbb',
-      valid: true,
+      valid: false,
```

Comment:
I think pen embed urls should be valid?

Reply:
Nope they shouldn’t be valid urls to create an embed for.
Just like the other transformers (see CodeSandbox or SoundCloud), we only support the normal URL’s to the content.

Reply:
oh okay, i will fix that now. well i thought its was fine because in getHTML we are replacing /pen/ to /embed/ soo either way it doesn't matter if we provide /pen/ or /embed/ link. no worries i will fix that now

Reply:
Let’s not support it (for now) for having consistency with the other transformers.

The point of using this plugin is really “Just copy the link you see in your browser onto a separate line in your Markdown file and it will transform it into an embedded viewer/player/...”

Maybe at a later point we can also support the embed-links, but I think that would be something we have to look at when we have more supported services so we have a better understanding of how everybody is having their embed-links.